docs/developer/shipping-zones.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Shipping Zones
+---
+
+## Introduction
+Shipping zones allow defining the cost of shipping in different areas of the world.
+Shipping zones contains shipping method that are available in given countries and
+channels.
+
+Specifying shipping zones is required to allow the selling of products to given countries.
```

Comment:
I guess it's only required for physical products, digital goods could be sold without shipping right?

```suggestion
Specifying shipping zones is required to allow the selling of physical products to given countries.
```